### DISCUSSION TITLE
Is there a way to make advanced search?

### DISCUSSION THREAD

**Discussion by noliveleger** [immich-app/immich#6844] [Q&A] [answered]
Hello there, 

First, congrats to all the team. The product is amazing.

I have been searching a way to search photos by date range, filesystem path, camera etc... but I could not figure out how do it? 
I found nothing relevant from https://immich.app/docs/features/search/ or from GH Discussions.
Is it implemented or the search only work with the auto-tagging?

**Comment by schuhbacca** [comment, marked as answer]
Search is being worked on currently you can see a preview in the 1.94 release notes.
